src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/models/AttributeGroup.ts

```diff
@@ -5,19 +5,8 @@ type AttributeGroupLabels = {
 type AttributeGroup = {
   code: string;
   sort_order: number;
-  attributes: string[];
   labels: AttributeGroupLabels;
-  permissions: {
-    view: string[];
-    edit: string[];
-  };
-  attributes_sort_order: {
-    [attribute: string]: number;
-  };
-  meta: {
-    id: number;
-  };
-  isDqiActivated?: boolean;
+  is_dqi_activated?: boolean;
```

Comment:
There is a reason that we have twice AttributeGroupe type (https://github.com/akeneo/pim-community-dev/pull/18901/files#diff-01e325759244da53067f06003654e84dc7adaf4402d1060dc849e5f65fe0cb41) ? 
```suggestion
  is_dqi_activated: boolean;
```